lib/configfiles/rhel7.xml

```diff
@@ -2,6 +2,38 @@
 <froxlor>
 	<distribution name="RHEL" version="7"
 		defaulteditor="/usr/bin/nano">
+      <defaults>
```

Comment:
that's more or less what I was thinking as distro-dependant setting-defaults. But why `name` and `category`? The settings are in `settinggroup`, `varname` and `value` - we should keep that equal to the rest of the system.

Reply:
agreed